content/docs/dvclive/api-reference/live/index.md

```diff
@@ -71,10 +71,12 @@ other metadata.
 
 ## Methods
 
+- `Live.get_step()`
 - `Live.log()`
 - `Live.log_image()`
+- `Live.log_param()`
+- `Live.log_params()`
 - `Live.log_plot()`
 - `Live.make_report()`
-- `Live.get_step()`
```

Comment:
Why was `get_step` moved up?

Reply:
I think it was made alphabetical, but good catch that it doesn't match the sidebar. If you're fine with it, we can update the sidebar also. Probably easier for users to find things that way than trying to guess some order of importance.

Reply:
Sure up to you. I was just pointing out a possible discrepancy I saw.